docs/appkit/react/wagmi/about/implementation.mdx

```diff
@@ -22,25 +22,25 @@ const projectId = 'YOUR_PROJECT_ID'
 const metadata = {
   name: 'AppKit',
   description: 'AppKit Example',
-  url: 'https://web3modal.com', // origin must match your domain & subdomain
+  url: 'https://example.com', // origin must match your domain & subdomain
   icons: ['https://avatars.githubusercontent.com/u/179229932']
-}
-
-export const networks = [mainnet, arbitrum]
+};
 
-// 3. Create Wagmi Adapter
-const wagmiAdapter = new WagmiAdapter({
+// 3. Set general configurations
+const generalConfig = {
+  projectId,
+  metadata,
+  networks: [mainnet, arbitrum],
   ssr: true,
-  networks,
-  projectId
-})
+};
+
+// 4. Create Wagmi Adapter
+const wagmiAdapter = new WagmiAdapter(generalConfig);
 
-// 4. Create modal
+// 5. Create modal
 createAppKit({
   adapters: [wagmiAdapter],
-  networks: [mainnet, arbitrum],
-  metadata,
-  projectId,
+  ...generalConfig,
```

Comment:
Let's not do this, please. It can be quite confusing for devs or new hackers building on top of AppKit. In the pursuit of reducing a few lines of code, we may end up confusing people even more. 

Ideally, all of our examples should explicitly show the general (or required/mandatory) params that should be passed.

Reply:
ok